Cargo.toml

```diff
@@ -18,4 +18,6 @@ wasmtime-wasi = "0.22"
 wasmtime-cache = "0.22"
 wasi-common = "0.22"
 clap = "2.33.3"
-tempfile = "3.2"
\ No newline at end of file
+tempfile = "3.2"
+bindle = "0.3"
```

Comment:
This should be changed to `bindle = { version = "0.3", default-features = false, features = ["client"]}` so we aren't pulling in all the server dependencies